src/tests/app.test.ts

```diff
@@ -22,17 +22,17 @@ test("default health check route should return 200", async (t) => {
 	t.is(response.body, JSON.stringify({ message: "OK" }));
 });
 
-test("documentation route should return 200", async (t) => {
-	const opts: InjectOptions = {
-		method: "GET",
-		url: {
-			pathname: "/documentation/static/index.html",
-			hostname: "localhost",
-			port: 8888,
-			protocol: "http",
-		},
-	};
-	const response = await t.context.server.inject(opts);
+// test("documentation route should return 200", async (t) => {
```

Comment:
Why was this unrelated test removed in this PR?

Reply:
The test did make all tests hang. Also I think the swagger UI is nice for development but should not land in production

Reply:
I understand. Would remove the test completely

Reply:
Yes. Will do